docs/cookbooks/index.rst

```diff
@@ -0,0 +1,157 @@
+Cookbooks
+=========
+
+The Cookbooks are collections of simple recipes that demonstrate good practices to accomplish
+common tasks. The examples are usually short answers to simple "How do I..." questions that go
+beyond simple API descriptions but also don't need a full guide to become clear.
+
+.. _evm_cookbook:
+
+EVM Cookbook
+~~~~~~~~~~~~
+
+.. _evm_cookbook_recipe_using_the_chain_object:
+
+Using the Chain object
+----------------------
+
+A "single" blockchain is made by a series of different virtual machines
+for different spans of blocks. For example, the Ethereum mainnet had
+one virtual machine for blocks 0 till 1150000 (known as Frontier),
+and another VM for blocks 1150000 till 1920000 (known as Homestead).
+
+The :class:`~eth.chains.chain.Chain` object manages the series of fork rules,
+after you define the VM ranges. For example, to set up a chain that would track
+the mainnet Ethereum network until block 1920000, you could create this chain
+class:
+
+.. doctest::
+
+  >>> from eth import constants, Chain
+  >>> from eth.vm.forks.frontier import FrontierVM
+  >>> from eth.vm.forks.homestead import HomesteadVM
+  >>> from eth.chains.mainnet import HOMESTEAD_MAINNET_BLOCK
+
+  >>> chain_class = Chain.configure(
+  ...     __name__='Test Chain',
+  ...     vm_configuration=(
+  ...         (constants.GENESIS_BLOCK_NUMBER, FrontierVM),
+  ...         (HOMESTEAD_MAINNET_BLOCK, HomesteadVM),
+  ...     ),
+  ... )
+
+Then to initialize, you can start it up with an in-memory database:
+
+.. doctest::
+
+  >>> from eth.db.backends.memory import MemoryDB
+  >>> from eth.chains.mainnet import MAINNET_GENESIS_HEADER
+
+  >>> # start a fresh in-memory db
+
+  >>> # initialize a fresh chain
+  >>> chain = chain_class.from_genesis_header(MemoryDB(), MAINNET_GENESIS_HEADER)
+
+.. _evm_cookbook_recipe_creating_a_chain_with_custom_state:
+
+Creating a chain with custom state
+----------------------------------
+
+While the previous recipe demos how to create a chain from an existing genesis header, we can
+also create chains simply by specifing various genesis parameter as well as an optional genesis
+state.
+
+.. doctest::
+
+  >>> from eth import constants
+  >>> from eth.chains.mainnet import MainnetChain
+  >>> from eth.db.backends.memory import MemoryDB
+  >>> from eth_utils import to_wei, encode_hex
+
+  >>> # Giving some funds to the zero address
+  >>> GENESIS_STATE = {
+  ...     constants.ZERO_ADDRESS: {
```

Comment:
Yeah, thought that, too as I wrote it. Should I use a random one (to not "promote" a specific one)

Reply:
Maybe pick a burnable private key like `b'unicornsrainbowsunicornsrainbows'` and then we can use that consistently to generate the *address* we use when working with either private keys or need a public address.

Reply:
>Maybe pick a burnable private key like b'unicornsrainbowsunicornsrainbows'

I feel stupid. Can you elaborate on what makes that key "burnable"?

Reply:
Only the fact that it's included here in plain text and thus, if anyone ever chose to use that key and stored $$ there it would likely be stolen by someone.